cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -2618,11 +2618,10 @@ pub(crate) fn emit(
             temp,
             dst_old,
         } => {
-            // FIXME: use real vregs for this seq.
```

Comment:
Ok, I might actually add another commit to this PR to avoid the hard coded registers... Just wanted to make sure this was on the right track. (Or I could open another PR, I guess).